downstream/modules/platform/proc-controller-set-up-prometheus.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-controller-set-up-prometheus"]
+
+= Setting up Prometheus
+
+To set up and use Prometheus, you must install Prometheus on a virtual machine or container. 
+
+For more information, see the link:https://prometheus.io/docs/introduction/first_steps/[Prometheus documentation].
+
+Use the following procedure to install Prometheus.
+
+.Procedure
+. In the Prometheus configuration file (typically `prometheus.yml`), specify a `<token_value>`, a valid username and password for an {ControllerName} user that you have created, and a `<controller_host>`.
++
+[NOTE]
+====
+Alternatively, you can provide an OAuth2 token (which can be generated at `/api/v2/users/N/personal_tokens/`). 
+By default, the configuration assumes a user with username=`admin` and password=`password`.
+====
++
+Using an OAuth2 Token, created at the `/api/v2/tokens` endpoint to authenticate prometheus with {ControllerName}, the following example provides a valid scrape configuration if the URL for your {ControllerName}'s' metrics endpoint is `https://controller_host:443/metrics`.
```

Comment:
Is this a typo: the URL for your {ControllerName}'s' - should the second quote be removed?